src/shared/apis/queryCache.ts

```diff
@@ -183,6 +194,33 @@ export const resetQueryCacheByQuery = (query: string): void => {
   queryCache.resetCacheByID(queryID)
 }
 
+export const getFromQueryCacheByQuery = (
```

Comment:
i'm too lazy to find where you copypasta'd this from. why the pasta, and not just update the original definition of the function?

Reply:
it's from one of the other query caches :(

Reply:
Updated. I should have pulled this out into its own function. Did that in the last commit